refine-invoice-genarator-blog (#1579)
pankod/refine: documentation/blog/2022-02-22-refine-invoice-genarator.md

---

> title: The Best Invoice Generator for Your Buisness Part I

Reviewer: ```suggestion
title: Title: Develop your Own Customizable Invoice Generator with Refine and Strapi | Part I
```

> title: Title: Develop your Own Customizable Invoice Generator with Refine and Strapi | Part I

Reviewer: ```suggestion
title: Develop your Own Customizable Invoice Generator with Refine and Strapi | Part I
```

> description: We will develop a fully customizable Invoice Maker Web application for your business using Refine and Strapi.

Reviewer: ```suggestion
description: Looking for an invoice generator? Try out Refine. With our custom interface, you can build your own invoice in minutes! Learn more here.
```
